src/Router.js

```diff
@@ -0,0 +1,51 @@
+import React, { lazy } from 'react'
+import { createBrowserHistory } from 'history'
+import { Navigate, Route, Routes } from 'react-router-dom'
+import { nanoid } from 'nanoid'
+const HomePage = lazy(() => import('./pages/Home'))
+const DetailPage = lazy(() => import('./pages/Detail'))
```

Comment:
please make your imports cleaner by separating them into sections as I requested few days ago

Reply:
i was fix that